images/users/lesalon.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you please run your SVG through [svgo](https://jakearchibald.github.io/svgomg/) to optimize it ?